binding.gyp

```diff
@@ -21,7 +21,7 @@
           'install_zmq': '<!(./build_libzmq.sh 2>&1 > zmq-build.log)',
           'xcode_settings': {
             'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
-            'MACOSX_DEPLOYMENT_TARGET': '10.6',
+            'MACOSX_DEPLOYMENT_TARGET': '10.7',
```

Comment:
I'm still on El Capitan and haven't changed anything apart from the node version.

We could also remove this line and it'll still work. But I haven't tested the prebuilts.